src/server/tran_server.hpp

```diff
@@ -74,6 +74,7 @@ class tran_server
 
     void disconnect_page_server ();
     bool is_page_server_connected () const;
+    void push_request_to (size_t idx, tran_to_page_request reqid, std::string &&payload);
```

Comment:
can have the same name, overloading does its job:
```suggestion
    void push_request (size_t idx, tran_to_page_request reqid, std::string &&payload);
```

Reply:
Done. But it makes `bind` hard :(. I've used `using`. Welcome to any suggestion.

Reply:
Yes, I did not anticipate this. And the new implementation of `active_tran_server::on_boot` is hard to parse/understand.
Would it be too much to as to revert is as it was initially?
I did not anticipate it would have this ugly repercussion.

Reply:
done!